Model/ResourceModel/Menu/Node.php

```diff
@@ -13,6 +13,29 @@ protected function _construct()
         $this->_init('snowmenu_node', 'node_id');
     }
 
+    protected function _afterSave(\Magento\Framework\Model\AbstractModel $object)
+    {
+        $connection = $this->getConnection();
+        $connection->delete('snowmenu_customer', ['node_id = ?' => $object->getNodeId()]);
+
+        $nodeCustomerGroups = $object->getData('customer_groups');
+        if ($nodeCustomerGroups && is_string($nodeCustomerGroups)) {
+            $nodeCustomerGroups = json_decode($nodeCustomerGroups);
```

Comment:
to serialize this we needed SerializerInterface, now to deserialize just using json_decode - what if serializerIntegrace changes implementation once again ?